entity-types/infra-mysqlnode/definition.yml

```diff
@@ -1,11 +1,41 @@
 domain: INFRA
 type: MYSQLNODE
+
+synthesis:
+  rules:
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0c50ef5a58683a214a509ee15c75279a6334d9b9/receiver/mysqlreceiver/metadata.yaml#L12C9-L12C17
+    - identifier: mysql.instance.endpoint # server.address:server.port
+      name: mysql.instance.endpoint 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: Metric
+        - attribute: metricName
+          prefix: mysql.
+        - attribute: instrumentation.provider
+          value: opentelemetry
+        # This filters only metrics coming from mysqlreceiver , given that metrics
+        # could differ between different runtime receivers.
+        - attribute: otel.library.name
+          value: otelcol/mysqlreceiver
+      tags:
```

Comment:
perhaps is worth to add the tags comming from the environment decoration like this https://github.com/newrelic/entity-definitions/blob/main/entity-types/infra-redisinstance/definition.yml#L31C28-L35